bin/ssh_scan

```diff
@@ -246,6 +246,11 @@ ssh_scan' to get the latest"
   end
 end
 
+# Limit scope of fingerprints DB to (per scan)
```

Comment:
This is mainly for performance/useability reasons

1.) Performance - the file will just get larger and larger as you scan more stuff, and doing look ups will take longer as more fingerprints are added
2.) If you're a tester and you scan customer A (192.168.0.0/24) and you go to customer B (192.168.0.0/24) and they have the same appliance with the shared host key, this could get confusing because we're merging colliding network space.